lib/smack/ConstantBVOps.cpp

```diff
@@ -31,6 +31,9 @@ unsigned getOpWidth(const Value *t) {
 }
 
 bool ConstantBVOps::runOnFunction(Function &f) {
+  if (!(SmackOptions::BitPrecise || SmackOptions::BitPrecisePointers))
```

Comment:
Sorry I missed this. Wouldn't be better if we move this guard to llvm2bpl.cpp so that this pass is independent of what SMACK is configured?

Reply:
That might be a better place. On the other hand, it documents something about this pass.